# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/nsSecCheckWrapChannel.h
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/base/nsSecCheckWrapChannel.h
--- a/netwerk/base/nsSecCheckWrapChannel.h	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/base/nsSecCheckWrapChannel.h	Tue Feb 12 16:08:25 2019 +0000
@@ -77,7 +77,7 @@
 };
 
 /* We define a separate class here to make it clear that we're overriding
- * Get/SetLoadInfo as well as AsyncOpen2() and Open2(), rather that using
+ * Get/SetLoadInfo as well as AsyncOpen() and Open(), rather that using
  * the forwarded implementations provided by NS_FORWARD_NSICHANNEL"
  */
 class nsSecCheckWrapChannel : public nsSecCheckWrapChannelBase {
@@ -85,8 +85,8 @@
   NS_IMETHOD GetLoadInfo(nsILoadInfo** aLoadInfo) override;
   NS_IMETHOD SetLoadInfo(nsILoadInfo* aLoadInfo) override;
 
-  NS_IMETHOD AsyncOpen2(nsIStreamListener* aListener) override;
-  NS_IMETHOD Open2(nsIInputStream** aStream) override;
+  NS_IMETHOD AsyncOpen(nsIStreamListener* aListener) override;
+  NS_IMETHOD Open(nsIInputStream** aStream) override;
 
   nsSecCheckWrapChannel(nsIChannel* aChannel, nsILoadInfo* aLoadInfo);
   static already_AddRefed<nsIChannel> MaybeWrap(nsIChannel* aChannel,